SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -143,7 +143,9 @@ def customise_DQM(process,pileup):
     process.DQMOfflineTracking.remove(process.TrackingDQMSourceTier0Common)
 
     # Pixel DQM needs to be updated for phase1
-    process.DQMOfflinePreDPG.remove(process.siPixelOfflineDQM_source)
+    old = process.siPixelOfflineDQM_source
+    process.load('DQM.SiPixelPhase1Common.SiPixelPhase1OfflineDQM_source_cff')
+    process.DQMOfflinePreDPG.replace(old, process.siPixelPhase1OfflineDQM_source)

```

Comment:
@schneiml At this point please don't add new customizations to `phase1TkCustoms` (it's being cleaned up). Use the era mechanism instead, and delete the line removing `siPixelOfflineDQM_source` from the `DQMOfflinePreDPG` sequence.

Essentially you probably want to use `eras.phase1Pixel.toReplaceWith()` in https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DQM/SiPixelCommon/python/SiPixelOfflineDQM_source_cff.py to replace the contents of `siPixelOfflineDQMSource`. See e.g. https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoTracker/IterativeTracking/python/iterativeTk_cff.py or https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoTracker/IterativeTracking/python/LowPtTripletStep_cff.py for practical examples (in tracking configuration).

Said that, for me these changes are fine for testing or RFC-PRs (like this one seems to be at the moment), but for integration an era-based solution is needed.